### PR TITLE
new poll and quick fix to setbg

### PR DESCRIPTION
**Pull request by timelessnesses**
**What kind of change does this PR introduce?**  
Feature

**Describe the changes proposed in the pull request**  
Adding a NEW and HOT poll command (with support of true false poll/multiple option poll and result command and event listener for poll if it's single or multiple choice poll)

**Does this PR introduce changes to the database?**
Did add SQL for poll

**Other information:**  
This is in beta. Please help me test this command.

### CODE REVIEW

File: bot/commands/polls/create_multiple_option.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+        "3️⃣",
+        "4️⃣",
+        "5️⃣",
+        "6️⃣",
+        "7️⃣",
+        "8️⃣",
+        "9️⃣",
+    ]
+
+    async def execute(self, arguments: list[str], message: discord.Message) -> None:
```
Comment (xsnowstorm) [inline bot/commands/polls/create_multiple_option.py:28]: I don‘t know if `arguments: list[str]` works because arguments[1] is a bool, not a string

Comment (timelessnesses) [reply bot/commands/polls/create_multiple_option.py:28]: it's old typing will likely fix this typing soon.

**Comment by timelessnesses** [comment]
merg3!!!!!

**Comment by timelessnesses** [comment]
conflict gaming

File: bot/commands/polls/create_multiple_option.py (hunk 1 above)
Comment (timelessnesses) [reply bot/commands/polls/create_multiple_option.py:28]: im too lazy for NamedTuple

**Pull request merged**